AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/StateMachine/ErrorMapping/SignInError+Helper.swift

```diff
@@ -78,6 +78,11 @@ extension HostedUIError: AuthErrorConvertible {
                 AuthPluginErrorConstants.hostedUISignOutURI.errorDescription,
                 AuthPluginErrorConstants.hostedUISignOutURI.recoverySuggestion)
 
+        case .signOutRedirectURI:
+            return .service(
```

Comment:
Shouldn't this (and probably `.signOutURI` as well) be mapped to a `.configuration` error?
Unless I'm missing something, it happens when there's no callback URI defined in the configuration file, right?